src/mc-fle2-payload-iev-private-v2.h

```diff
@@ -73,6 +73,13 @@ bool mc_FLE2IndexedEncryptedValueV2_add_K_Key(_mongocrypt_crypto_t *crypto,
 const _mongocrypt_buffer_t *mc_FLE2IndexedEncryptedValueV2_get_ClientValue(const mc_FLE2IndexedEncryptedValueV2_t *iev,
                                                                            mongocrypt_status_t *status);
 
+uint8_t mc_FLE2IndexedEncryptedValueV2_get_edge_count(const mc_FLE2IndexedEncryptedValueV2_t *iev,
+                                                      mongocrypt_status_t *status);
+
+const _mongocrypt_buffer_t *mc_FLE2IndexedEncryptedValueV2_get_edge(const mc_FLE2IndexedEncryptedValueV2_t *iev,
```

Comment:
Ideally, I'd like the parser to pull out the edge components individually into a sub-struct, so something like:

```c
typedef struct _mc_FLE2EncryptedValueV2_Edge_t {
    _mongocrypt_buffer_t encryptedCount;
    _mongocrypt_buffer_t tag;
    _mongocrypt_buffer_t encryptedZeros;
} mc_FLE2EncryptedValueV2_Edge_t;
```

These are each 32 bytes (of the 96 byte metadatalen total).

This should also return by out-param rather than return value to maintain consistency with other mongocrypt APIs, e.g.

```c
bool mc_FLE2IndexedEncryptionValueV2_get_edge(const mc_FLE2IndexedEncryptedValueV2_t *iev,
            mc_FLE2EncryptedValueV2_Edge_t* out,
            const uint8_t edge_index,
            mongocrypt_status_t *status);
```

The struct might need to be in its own header with `_init` and `_cleanup` methods matching the style of other types.